Project.toml

```diff
@@ -0,0 +1,6 @@
+name = "Cassette"
+uuid = "7057c7e9-c182-5462-911a-8362d720325c"
+version = "0.2.4"
+
+[compat]
+julia = ">= 1.0"
```

Comment:
@oxinabox like so?

Reply:
Probable just `1`
If ever there was software that will break in Julia 2.0 it is Cassette.

On that basis
`~1.0, ~1.1, ~1.2, ~1.3`
might even be reasonable.

Reply:
Re: `~1.0, ~1.1, ~1.2, ~1.3` I see your point, but on the other hand Cassette has been mostly compatible across versions.